src/devices/GamepadMappings.js

```diff
@@ -131,10 +174,14 @@ let windowsMixedReality = {
 };
 
 let GamepadMappings = {
+  'Daydream Controller': daydream,
+  'Gear VR Controller': samsungGearVR,
   'Oculus Go Controller': oculusGo,
   'Oculus Touch (Right)': oculusTouch,
   'Oculus Touch (Left)': oculusTouch,
   'OpenVR Gamepad': openVr,
+  'Spatial Controller (Spatial Interaction Source) 045E-065A': windowsMixedReality,
+  'Spatial Controller (Spatial Interaction Source) 045E-065D': samsungOdyssey,
   'Windows Mixed Reality (Right)': windowsMixedReality,
```

Comment:
I'm not sure, but I don't think this is reported anywhere?

Reply:
Er to be clear, I mean line 185

Reply:
That was what Chrome was reporting with WebVR previously. Included for completeness.